### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by tomrf1**

### CODE REVIEW

File: docs/channel-tests.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                                     +--------+
++---------------+  PUT  +----+  GET  |        |
+|               +-------> S3 <-------+        |
+|               |       +----+       |        | GET
+| admin-console |                    | Fastly <----+theguardian.com
+|               +-------------------->        |
+|               |       PURGE        |        |
++---------------+                    |        |
+                                     +--------+
+```
```
Comment (TomPretty) [inline docs/channel-tests.md:56]: This is great!

**Review by TomPretty — APPROVED** [review]
Looks great!

File: docs/channel-tests.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#### /liveblog-epic-tests
+Configures the liveblog epic.
+
+Note - this is the only channel that is still rendered 'natively' by frontend, rather than served by support-dotcom-components.
+
+Whilst this is still the case, the client has to fetch the tests json file directly. This file is served by Fastly, and the origin is in the gu-contributions-public S3 bucket.
+
+_Caching of liveblog-epic-tests.json_
+
+(This can go away once we've migrated frontend to fetch liveblog epics from support-dotcom-components.)
```
Comment (paulbrown1982) [inline docs/channel-tests.md:37]: _After_ the election 😉

**Review by paulbrown1982 — APPROVED** [review]
👍

**Pull request merged**